token/cli/src/output.rs

```diff
@@ -257,6 +259,20 @@ impl fmt::Display for CliTokenAccount {
             writeln!(f, "* Please run `spl-token gc` to clean up Aux accounts")?;
         }
 
+        if self.has_permanent_delegate {
+            writeln!(f)?;
+            writeln!(
+                f,
+                "* {} ",
+                style("This account's mint has a permanent delegate!").bold()
```

Comment:
nit: maybe keep it simpler?
```suggestion
                style("This token mint has a permanent delegate!").bold()
```